index.xml

```diff
@@ -582,7 +583,8 @@ content-length: 18</artwork>
     </figure>
 Note that the '\n' symbols above are included to demonstrate where the new
 line character should be inserted. There is no new line on the final line of
-the signing string.
+the signing string. The example displays each header on a new line for better
+legibility.
```

Comment:
or perhaps "Each HTTP header above is displayed on a new line to provide better readability of the example."